tailcfg/tailcfg.go

```diff
@@ -183,7 +183,9 @@ func (emptyStructJSONSlice) UnmarshalJSON([]byte) error { return nil }
 type Node struct {
 	ID       NodeID
 	StableID StableNodeID
-	Name     string // DNS
+
+	// Name is the FQDN of the node. It has a trailing dot.
```

Comment:
you dropped the word DNS in here. Add it back, and maybe give a full example value in quotes while here?

Reply:
Done